cypress/features/regression/dialog.feature

```diff
@@ -66,4 +66,15 @@ Feature: Dialog component
   @positive
   Scenario: Disable background UI
     When I open default "Dialog" component in noIFrame with "dialog" json from "commonComponents" using "enableBackgroundUIFalse" object name
-    Then Background UI is disabled in NoIFrame
\ No newline at end of file
+    Then Background UI is disabled in NoIFrame
+
+  # Sticky form footer tests
```

Comment:
do we need this comment ?

Reply:
yes, we do, because I've moved those tests from other feature file